### PR TITLE
Describe setting coredns server in nameserver resolv chain

### PR DESCRIPTION
**Pull request by shashidharatd**
This PR describes how to add CoreDNS server to nameserver resolv chain while adding k8s cluster to federation.

/assign @madhusudancs 
/cc @csbell 

This is supposed to go in 1.7 milestone. I am not sure how to set the milestone. @madhusudancs, can you please help me with this.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3997)

### CODE REVIEW

**Comment by madhusudancs** [comment]
@chenopis this looks good to me. Could you please take a look?

File: docs/tasks/federation/set-up-coredns-provider-federation.md [hunk 1]
```diff
@@ -97,16 +97,22 @@ coredns-provider.conf has below format:
...
  - `zones` is the federation domain for which CoreDNS is authoritative and is same as --dns-zone-name flag of `kubefed init`.
+ - `coredns-endpoints` is the endpoint to access coredns server. This is an optional parameter introduced from v1.7 onwards.
 
 *Note: middleware.etcd.zones in CoreDNS configuration and --dns-zone-name
 flag to kubefed init should match.*
 
 
 ## Setup CoreDNS server in nameserver resolv.conf chain
 
+*Note: Following section applies only to versions prior to v1.7 and
```
Comment (chenopis) [inline docs/tasks/federation/set-up-coredns-provider-federation.md]: Suggested edit (add "the", "of", and closing * to end italics):

*Note: The following section applies only to versions prior to v1.7 and will be automatically taken care of if the `coredns-endpoints` parameter is configured in `coredns-provider.conf` as described in section above.*

**Comment by chenopis** [comment]
Sorry for the delay.

File: docs/tasks/federation/set-up-coredns-provider-federation.md (hunk 1 above)
Comment (madhusudancs) [inline docs/tasks/federation/set-up-coredns-provider-federation.md]: @chenopis addressed this in a follow up PR - https://github.com/kubernetes/kubernetes.github.io/pull/4157. Can you please merge this and the follow up?

**Pull request merged**